src/openforms/prefill/contrib/haalcentraal/tests/test_plugin.py

```diff
@@ -107,8 +101,224 @@ def test_get_prefill_values_http_404(self, m):
         expected = {}
         self.assertEqual(values, expected)
 
-    def test_get_available_attributes(self):
+
+class HaalCentraalPrefillApiV2Test(TestCase):
```

Comment:
See the earlier remark about adding a base class for tests, as it looks like the actual tests are identical.

It's easy to add a regression test for V1 and then forget to add it for V2 or vice versa :thinking: